samples/org_website_samples/queue_sample.go

```diff
@@ -35,8 +35,7 @@ func queueSampleRun() {
 	queue.PollWithTimeout(5, time.Second)
 	//Indefinitely blocking Operations
 	queue.Put("yetanotheritem")
-	item, _ := queue.Take()
-	fmt.Println(item)
+	fmt.Println(queue.Take())
```

Comment:
This is also going to print the error, if there isnt error it will print `<nil>`

Reply:
I think that's Ok